Data.json

```
@@ -93,5 +93,10 @@
     "name": "Nikhil Dhariwal",
     "username": "404Nikhil",
     "avatar": "https://github.com/404Nikhil.png"
+  },
+  {
+    "name": "Hritesh Goldar",
+    "username": "hritesh94",
+    "avatar": "https://avatars.githubusercontent.com/u/89804314?s=400&v=4" 
```

My suggestion is to change your GitHub profile photo to same as in your twitter. And change the below link :point_down: 
```suggestion
    "avatar": "https://github.com/hritesh94.png" 
```
Or
you can directly use your twitter profile photo like :point_down: 
```suggestion
    "avatar": "https://pbs.twimg.com/profile_images/1572963702827257858/ETu4UReH_400x400.jpg" 
```